style.css

```diff
@@ -0,0 +1,649 @@
+}
+
+.left {
+  margin-left: 50%;
+}
+
+.supporters img {
+  width: 100px;
+  height: 70px;
+}
+
+.foot-head {
+  color: #fff;
+  text-align: center;
+  padding-top: 20px;
+}
+
+.footarea {
+  background-color: rgb(70, 69, 69);
+}
+
+.idea {
+  display: flex;
+}
+
+.logo-img2 img {
+  width: 40px;
+  height: 40px;
+}
+
+.last-container {
+  display: flex;
+  justify-content: space-around;
+  margin: 20px;
+}
+
+.logo-text {
+  line-height: 17px;
+}
+
+.foot {
+  display: block;
+}
+
+.foot-text {
+  line-height: 15px;
+  font-size: 14px;
+}
+
+footer,
+.desktop,
+.navigation,
+.see {
+  display: none;
+}
+
+.spaeaker-card:nth-child(3),
+.spaeaker-card:nth-child(4),
+.spaeaker-card:nth-child(5),
+.spaeaker-card:nth-child(6) {
+  display: none;
+}
+
+#menupopup {
+  width: 100%;
+  height: 100vh;
+  display: none;
+  background-color: rgb(186, 61, 61);
+  position: absolute;
+  z-index: 1;
+  top: 0;
+}
+
+.menupopup img {
+  width: 12px;
+  height: 12px;
+  position: absolute;
+  top: 2%;
+  right: 5%;
+}
+
+.menupopup ul a {
+  padding-top: 26px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.menupopup ul {
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  font-weight: 600;
+  font-size: 32px;
+  padding: 15% 5%;
+}
+
+#chaina {
+  display: none;
+}
+
+@media (min-width: 867px) {
+  .welcome-msg-2 p,
+  .humburger-2,
+  .humburger {
+    display: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .navigation {
+    display: block;
+  }
+
+  .spaeaker-card:nth-child(3),
+  .spaeaker-card:nth-child(4),
+  .spaeaker-card:nth-child(5),
+  .spaeaker-card:nth-child(6) {
+    display: block;
+  }
+
+  .nav-bar {
+    background-color: #000;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .nav-list ul {
+    display: flex;
+    justify-content: right;
+    padding-right: 80px;
+  }
+
+  .nav-list ul li {
+    color: #fff;
+    margin: 10px 15px;
+  }
+
+  .navbar-2 {
+    display: flex;
+    justify-content: space-between;
+    background-color: #fff;
+  }
+
+  .idea-2 {
+    margin: 20px 50px;
+  }
+
+  .nav-list-2 {
+    padding-right: 10%;
+  }
+
+  .nav-list-2 ul {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-list-2 ul li {
+    margin: 10px 10px;
+  }
+
+  .last-list a,
+  .first-list a {
+    color: rgb(186, 61, 61);
+  }
+
+  .last-list {
+    border: 5px solid rgb(186, 61, 61);
+    height: 50px;
+    color: rgb(186, 61, 61);
+    width: 150px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .welcome-msg {
+    height: 40vh;
+  }
+
+  .setting {
+    height: 20vh;
+    padding: 20px 15% 0;
+  }
+
+  .description {
+    width: 40%;
+    height: 50%;
+    margin: 70px 16% 0;
+  }
+
+  .welcome-msg h1 {
+    font-size: 50px;
+    padding: 20px 0 15% 15%;
+  }
+
+  .welcome-msg p {
+    padding: 10% 15% 0;
+    font-size: 30px;
+  }
+
+  .outline-items {
+    display: grid;
+    grid-template-columns: 20% 20% 20% 20% 20%;
+    margin: 5% 20px;
```

Comment:
- [ ] It will be better to add `10%` margin to right and left to make your design has the same spacing as the Behance. ` margin: 5% 10%;`